.github/workflows/post-release.yml

```diff
@@ -0,0 +1,100 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+name: post-release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  upload_uberjars:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - id: 'auth'
+      name: 'Authenticate to Google Cloud'
+      uses: 'google-github-actions/auth@v1.0.0'
+      with:
+        workload_identity_provider: ${{ secrets.PROVIDER_NAME }}
+        service_account: ${{ secrets.SERVICE_ACCOUNT }}
+
+    - name: Set up Cloud SDK
+      uses: google-github-actions/setup-gcloud@v1
+
+    - name: Cloud Build
+      id: cloudbuild
+      env:
+        RELEASE_PROJECT: '${{ secrets.RELEASE_PROJECT }}'
```

Comment:
it's a different project, so AFAIK we'll need to grant permissions to the WIF service account to run builds and push to GCS. I'm not sure what APIs need to be enabled on the WIF project though, since ideally everything would run on the release project.

Reply:
As far as I know the default for whenever the WIF SA is used outside the WIF project is that it will use the services in the separate project but always use the API and quotas from the WIF project. For example, the tests use the Cloud SQL Admin API quota from the WIF project even though the Cloud SQL instances are in the testing project.